joern-cli/frontends/kotlin2cpg/src/main/scala/io/joern/kotlin2cpg/ast/AstForPrimitivesCreator.scala

```diff
@@ -211,11 +212,13 @@ trait AstForPrimitivesCreator(implicit withSchemaValidation: ValidationMode) {
   }
 
   def astForImportDirective(directive: KtImportDirective): Ast = {
-    val isWildcard = directive.getLastChild.getText == Constants.wildcardImportName || directive.getImportedName == null
+    val importedAs = Try(directive.getImportedName.getIdentifier).toOption.getOrElse("")
```

Comment:
Why are we not using `directive.getLastChild.getText`? Is this the full name?

Reply:
Yes, this is coming out as fullname. Example - `com.firebase.ui.auth.AuthUI`

Reply:
I believe `importedAs` should be `AuthUI` and then `importedEntity` should be `com.firebase.ui.auth.AuthUI`

Reply:
Yes, it is like that

Reply:
Ok cool 👍

Reply:
I think we shouldn't use a `""` if `Try(directive.getImportedName.getIdentifier)` fails. We can leave `importedAs` as a `None` in these cases.

Reply:
Then the next line can become something like

```scala
val isWildcard = importedAs.exists(_ == Constants.wildcardImportName) || directive.getImportedName == null
```

Reply:
ok, noted

Reply:
@DavidBakerEffendi Have addressed it here - 6d3480e7504c2f0c45a001c17f60f0131b0f0c12